### PR TITLE
Added missing HDRP debug menu documentation

### PR DESCRIPTION
**Pull request by adrien-de-tocqueville**
### Purpose of this PR

Update pass on the Debug menu documentation
Still missing section about probe volumes, but there's kinda no doc for that

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Documentation~/Post-Processing-Propagating-NaNs.md [hunk 1]
```diff
@@ -40,11 +40,14 @@ If you are unable to fix the source of the NaN/Inf values, [HDRP Cameras](HDRP-C
...
 To find the root cause of a NaN/Inf, HDRP includes a debug mode which displays pixels that contain NaNs/Infs in a recognizable color. To use this debug mode:
 
-1. Open the Render Pipelines Debug window (menu: **Window > Render Pipeline > Render Pipeline Debug**).
+1. Open the Render Pipeline Debug window (menu: **Window > Render Pipeline > Render Pipeline Debug**).
 2. Go to **Rendering** and set **Fullscreen Debug Mode** to **NanTracker**.
 
 This helps you to see if there are actually NaNs/Infs on screen and which material causes them. However, if you need more information, such as which particular draw call causes the issue, you can use frame debugging tools such as [RenderDoc](https://renderdoc.org/). For information on how to use RenderDoc to capture frames in Unity, see [RenderDoc integration](https://docs.unity3d.com/Manual/RenderDocIntegration.html).
 
+A common situation that leads to NaNs is when a mesh is imported with ill-defined normals, like normals equal to the zero vector.
+These normals can be found using one of the normal visualization modes in the [**Material** panel](Render-Pipeline-Debug-Window.html#material-panel) of the Render Pipeline Debug window
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Post-Processing-Propagating-NaNs.md:49]: To find these normals, you can use one of the normal visualization modes in the [**Material** panel](Render-Pipeline-Debug-Window.html#material-panel) of the Render Pipeline Debug window.

File: com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Debug-Window.md [hunk 2]
```diff
@@ -40,6 +41,20 @@ To display the current active item independently of the debug window:
...
 
+<a name="RenderGraphPanel"></a>
+
+## Render Graph panel
+
+The **Render Graph** panel has tools that you can use to debug the [Render Graph](https://docs.unity3d.com/2020.2/Documentation/Manual/render-graph.html) used by HDRP.
+
+| **Debug Option**  | **Description**                                              |
+| ----------------- | ------------------------------------------------------------ |
+| **Clear Render Targets at creation**  | Enable the checkbox to clear render targets the first time they are used. |
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Debug-Window.md:52]: Enable the checkbox to make the render graph system clear render targets the first time it uses them.

File: com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Debug-Window.md [hunk 3]
```diff
@@ -40,6 +41,20 @@ To display the current active item independently of the debug window:
...
+| **Disable Pass Culling**              | Enable the checkbox to render passes which have no impact on the final render. |
+| **Immediate Mode**                    | Enable the checkbox to have Render Graph evaluate passes immediately after they are created. |
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Debug-Window.md:54]: Enable the checkbox to make the render graph system evaluate render passes immediately after it creates them.

Comment (JordanL8) [reply com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Debug-Window.md:54]: What happens when the checkbox is disabled? Is this when they are evaluated at the end of all the render pass declaration when the graph compiles and executes?

File: com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Debug-Window.md [hunk 4]
```diff
@@ -109,6 +127,7 @@ The **Lighting** panel has tools that you can use to visualize various component
 | **Clear Shadow Atlas**  | Enable the checkbox to clear the shadow atlas every frame.   |
 | **Range Minimum Value** | Set the minimum shadow value to display in the various shadow debug overlays. |
 | **Range Maximum Value** | Set the maximum shadow value to display in the various shadow debug overlays. |
+| **Log Cached Shadow Atlas Status** | Press the button to log in the console the list of lights placed in the the cached shadow atlas. |
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Debug-Window.md:130]: Press the button to log in to the Console the list of lights placed in the cached shadow atlas.

File: com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Debug-Window.md [hunk 5]
```diff
@@ -117,28 +136,28 @@ The **Lighting** panel has tools that you can use to visualize various component
...
 | **- Area Lights**                     | Enable the checkbox to see Area Lights in your Scene. Disable this checkbox to remove Aera Lights from your Scene's lighting. |
 | **- Reflection Probes**               | Enable the checkbox to see Reflection Probes in your Scene. Disable this checkbox to remove Reflection Probes from your Scene's lighting. |
+| **Exposure**                          | Allows you to select an [Exposure](Override-Exposure.md) debug mode to use. |
+| **- Debug Mode**                      | Use the drop-down to select a debug mode. See [Exposure](Override-Exposure.md#exposure-debug-modes) documentation for more information. |
+| - **Show Tonemap curve**              | Enable the checkbox to overlay the tonemap curve to the histogram debug view.<br/>This property only appears when you select **HistogramView** from **Debug Mode**. |
+| **- Center Around Exposure**          | Enable the checkbox to overlay the center the histogram around the current exposure value.<br/>This property only appears when you select **HistogramView** from **Debug Mode**. |
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Debug-Window.md:142]: Enable the checkbox to center the histogram around the current exposure value.<br/>This property only appears when you select **HistogramView** from **Debug Mode**.

File: com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Debug-Window.md [hunk 6]
```diff
@@ -117,28 +136,28 @@ The **Lighting** panel has tools that you can use to visualize various component
...
+| **- Display RGB Histogram**           | Enable the checkbox to display the Final Image Histogram as an RGB histogram instead of just luminance.<br />This property only appears when you select **FinalImageHistogramView** from **Debug Mode**. |
+| **- Display Mask Only**               | Enable the checkbox to display only the metering mask in the picture-in-picture, if unchecked the mask will displayed after weighting the scene color instead. <br />This property only appears when you select **MeteringWeighted** from **Debug Mode**. |
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Debug-Window.md:144]: Enable the checkbox to display only the metering mask in the picture-in-picture. When disabled, the mask displays after weighting the scene color instead. <br />This property only appears when you select **MeteringWeighted** from **Debug Mode**.

**Review by JordanL8 — CHANGES_REQUESTED** [review]
Rewrote some bits

File: com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Debug-Window.md (hunk 3 above)
Comment (adrien-de-tocqueville) [reply com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Debug-Window.md:54]: Disabled is default behavior. After creation, the unused passes are culled and the remaining one are executed

**Review by JordanL8 — APPROVED** [review]

**Pull request merged**